x-pack/plugins/infra/public/utils/formatters/data.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { InfraWaffleMapDataFormat } from '../../lib/lib';
+import { formatNumber } from './number';
+
+const getPowerFromValue = (val: number, base: number, labels: string[]) => {
+  for (let i = 0; i < labels.length; i++) {
+    if (val % Math.pow(base, i) === val) {
+      return i - 1;
+    }
+  }
+  return 0;
+};
+/**
+ * The labels are derived from these two Wikipedia articles.
+ * https://en.wikipedia.org/wiki/Kilobit
+ * https://en.wikipedia.org/wiki/Kilobyte
+ */
+const LABELS = {
+  [InfraWaffleMapDataFormat.bytesDecimal]: ['', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
+  [InfraWaffleMapDataFormat.bytesBinaryIEC]: [
+    '',
```

Comment:
Should these be 'B', 'b', 'bit' and so on respectively instead of empty? Otherwise this leads to inconsistent unit labeling:

![image](https://user-images.githubusercontent.com/973741/44862602-53b31400-ac7b-11e8-805c-608a98f36ad0.png)

Reply:
Cool... will make the change